### PR TITLE
Source Zendesk Support: Convert `tickets.custom_fields` values to string

### PR DESCRIPTION
**Pull request by grubberr**
Signed-off-by: Sergey Chvalyuk <grubberr@gmail.com>

## What
Fix for https://github.com/airbytehq/oncall/issues/320
"tickets.custom_fields" can be 
```
"custom_fields": [
                {
                    "id": 360023382300,
                    "value": null
                },
                {
                    "id": 360004841380,
                    "value": "customer_tickets____transfer_in____origo_customer_action_required__rejection_"
                },
                {
                    "id": 360022469240,
                    "value": "5"
                },
                {
                    "id": 360023712840,
                    "value": false
                },
            ],
```

value(s) can be "boolean" type but current schema:

```
{
  "custom_fields": {
    "type": ["null", "array"],
    "items": {
      "properties": {
        "id": {
          "type": ["null", "integer"]
        },
        "value": {
          "type": ["null", "string"]
        }
      },
      "type": ["null", "object"]
    }
  }
}
```

## How
We need to convert "boolean" to "string" for conform schema

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [x] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [x] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [x] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by arsenlosenko — APPROVED** [review]

File: airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py [hunk 1]
```diff
@@ -475,6 +475,13 @@ class Tickets(SourceZendeskIncrementalExportStream):
 
     response_list_name: str = "tickets"
 
+    def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py:478]: any reason not to use the TypeTransformer [here](https://docs.airbyte.com/connector-development/cdk-python/schemas#type-transformation)?

Comment (grubberr) [reply airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py]: Changed to use TypeTransformer!

**Review by sherifnada — APPROVED** [review]

**Pull request merged**